### PR TITLE
Fix the bare-metal CI

### PR DESCRIPTION
**Pull request by ustiugov**
## Summary

Fixes to the CI runner deployer script to set up the vHive CI. 

## Implementation Notes :hammer_and_pick:

* 

## External Dependencies :four_leaf_clover:

* 

## Breaking API Changes :warning:

* 

*Simply specify none (N/A) if not applicable.*

### CODE REVIEW

**Comment by ustiugov** [comment]
Fixed CRI runners need to run as root (because NTU VMs have only root by default)

Still need to debug the CRI runners, integ runners seems to work fine. @CuriousGeorgiy can you please review? Also need to remove the restart option entirely from all the runners (assume the user reboots or re-creates the host VMs manually, it's easy). 

@shyamjesal can you please take a look at the CRI runners and help us understand what can be the issue with them?

**Comment by ustiugov** [comment]
Another problem for @CuriousGeorgiy:
```
Failed: error: exists /etc/systemd/system/actions.runner.vhive-serverless-vhive.instance-ietu8h-2.service
Creating launch runner in /etc/systemd/system/actions.runner.vhive-serverless-vhive.instance-ietu8h-2.service 
FATA[0084] Failed to setup runner on root@10.0.0.22: Process exited with status 1 
exit status 1
```

I cannot clean and re-init the runner after reboot, so it becomes unoperational.

File: scripts/github_runner/deploy_runners.go [hunk 1]
```diff
@@ -147,14 +147,14 @@ func deployRunner(host string, runnerConf RunnerConf, deployerConf *DeployerConf
...
 
-		setupCmd = fmt.Sprintf("./scripts/github_runner/setup_integ_runners.sh %d %s %s %s", runnerConf.Num,
+		setupCmd = fmt.Sprintf("cd vhive && ./scripts/github_runner/setup_integ_runners.sh %d %s %s %s", runnerConf.Num,
 			deployerConf.GhOrg, deployerConf.GhPat, restart)
 	default:
 		log.Fatalf("Invalid runner type: '%s', expected 'cri' or 'integ'", runnerConf.Type)
 	}
 
 	log.Debugf("Setting up runner on %s@%s", deployerConf.HostUsername, host)
-	out, err = client.Exec(fmt.Sprintf("cd vhive && %s", setupCmd))
```
Comment (CuriousGeorgiy) [inline scripts/github_runner/deploy_runners.go:157]: What is the point of this change?

Comment (ustiugov) [reply scripts/github_runner/deploy_runners.go:157]: Readability of the code.

**Pull request merged**